WordPress/src/main/AndroidManifest.xml

```diff
@@ -214,6 +214,15 @@
                 <action android:name="android.intent.action.MAIN" />
             </intent-filter>
         </activity>
+
+        <!-- Story composer activity -->
+        <activity
+            android:name=".ui.stories.StoryComposerActivity"
+            android:label="@string/app_name"
+            android:screenOrientation="portrait"
```

Comment:
I'm a bit worried by this because it's the only screen that forces portrait mode, but I doubt there is a short term solution for that. I'm just curious if you have long a term plan to support landscape mode?

Reply:
Good question, we've discussed that in the team but the Stories format (stories composition UI) seems to have been largely adopted as a portrait-mode only kind of experience on phones. Deferring to @megsfulton for further info.
But apart from that,  do you see any technical problems I may be overseeing @maxme ?

Reply:
>  do you see any technical problems I may be overseeing @maxme ?